### PR TITLE
Made user selected course names override original course names when available

### LINKED ISSUE #88: Respect course nicknames
Use user selected course nicknames instead of course code when applicable. Just modify [useCourseNames](../blob/main/src/pages/Content/modules/hooks/useCourseNames.ts) to fetch nicknames and apply those that exist.

### PR DESCRIPTION
**Pull request by rk234**
Closes #88 

It looks like the canvas API will add an `original_name` field to courses that have a custom nickname assigned, so I modified `getCourseNames` to check for the existence of that field and return the user's preferred course name instead. Let me know if you would prefer a different approach.

### CODE REVIEW

File: package-lock.json
Comment (jtcheng26) [inline package-lock.json:1]: Prefer to remove these changes from the commit

File: src/pages/Content/modules/hooks/useCourseNames.ts
Comment (jtcheng26) [inline src/pages/Content/modules/hooks/useCourseNames.ts:1]: Just checking `course.original_name ? course.name as string : course.course_code as string` would be cleaner

**Review by jtcheng26 — CHANGES_REQUESTED** [review]
Looks good mostly, I trust you tested it reasonably

**Comment by rk234** [comment]
Thanks, everything should have been addressed now after some wrangling with git.

**Review by jtcheng26 — APPROVED** [review]

**Comment by rk234** [comment]
done

**Review by jtcheng26 — APPROVED** [review]

**Pull request merged**